Artesian/_Query/ActualQuery.py

```diff
@@ -1,59 +1,147 @@
+from Artesian import _ClientsExecutor
+from Artesian._ClientsExecutor import RequestExecutor
 from Artesian._Query.Query import _Query
 from Artesian._Query.QueryParameters.ActualQueryParameters import ActualQueryParameters
 from Artesian._Query.Config.ExtractionRangeConfig import ExtractionRangeConfig
 from Artesian._Query.Config.Granularity import Granularity
 from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
 import urllib
+from typing import List
+from __future__ import annotations
+
 class _ActualQuery(_Query):
     __routePrefix = "ts"
-    def __init__(self, client, requestExecutor, partitionStrategy):
+    def __init__(self, client: _ClientsExecutor, requestExecutor: RequestExecutor, partitionStrategy: DefaultPartitionStrategy) -> None:
+        """ Inits _ActualQuery
+         
+            Args:          
+                client 
+
+                requestExecutor    
+
+                partitionStrategy. """
         queryParameters = ActualQueryParameters(None,ExtractionRangeConfig(), None, None, None, None, None) 
         _Query.__init__(self, client, requestExecutor, queryParameters)
         self.__partition= partitionStrategy
 
-    def forMarketData(self, ids):
+    def forMarketData(self, ids: List[int]) -> _ActualQuery:
+        """ Set the list of marketdata to be queried.
+
+            Args:
```

Comment:
in _GMEPOfferQuery 'Returns' section is present, here it isn't.
please align: either Returns in present in all methods or isn't. pick one.